contents.json

```diff
@@ -2972,6 +2972,11 @@
       "category": "ui",
       "description": "Simple parallax header for UIScrollView.",
       "homepage": "https://github.com/maxep/MXParallaxHeader"
+    }, {
+      "title": "Arale",
+      "category": "ui",
+      "description": "A custom stretchable header view for UIScrollView or any its subclasses with UIActivityIndicatorView support. Built for iOS 10 and later.",
```

Comment:
@ZulwiyozaPutra could you remove "Built for iOS 10 and later." from the description?